Makefile

```diff
@@ -742,10 +742,6 @@ hostname.o: \
 compile hostname.c substdio.h subfd.h substdio.h readwrite.h exit.h
 	./compile hostname.c
 
-install:
-	echo './instpackage && ./instchown' > install
```

Comment:
What about keeping:
```
install:
	./instpackage
```

I do not think that anyone who discovered qmail got used to run `make install` for regenerate this simple shell script, and rather have the `install` target called as a dependency of `make it`.

So we might not break any expectation through your deletion.  Do I open a separate branch for this?